### PR TITLE
Error handling

### PR DESCRIPTION
**Pull request by kuol**
Basic error handling for user inputs.

### CODE REVIEW

File: NAMESPACE [hunk 1]
```diff
@@ -4,6 +4,10 @@ S3method(constructModel,file_mode)
 S3method(constructModel,manual_mode)
 S3method(constructModel,matrix_mode)
 export(AlteryxSolve)
+export(checkInputs)
```
Comment (ramnathv) [inline NAMESPACE]: We dont need this anymore. Can you remove it?

File: R/assertions.R [hunk 2]
```diff
@@ -0,0 +1,56 @@
+#' Check input quality
+#'
+#'
+#' @export
+#' @import assertthat
+checkInputs <- function(x, ...){
+  UseMethod('checkInputs')
+}
+
```
Comment (ramnathv) [inline R/assertions.R]: Not using S3 methods anymore. Remove this.

**Comment by kuol** [comment]
sorry, late for this...

**Comment by ramnathv** [comment]
No problems. It is just housekeeping, which we can do later.

**Pull request merged**